Translations/WinMerge/French.po

```diff
@@ -368,7 +374,7 @@ msgstr "Décompression &automatique"
 #: Merge.rc:714B45FD
 #, c-format
 msgid "&Edit with Unpacker..."
-msgstr ""
+msgstr "&Éditer avec le décompresseur..."
```

Comment:
Anglicisme: should be "Modifier" (and there is probably the same mistake at other places)